t/t2204-job-info.t

```diff
@@ -0,0 +1,109 @@
+test_expect_success 'flux job eventlog works on multiple entries (active)' '
+        jobid=$(flux jobspec --format json srun -N1 hostname | flux job submit) &&
+        kvsdir=$(flux job id --to=kvs-active $jobid) &&
+	flux kvs eventlog append ${kvsdir}.eventlog foo &&
+	flux job eventlog $jobid >eventlog_b.out &&
+	grep -q submit eventlog_b.out &&
+	grep -q foo eventlog_b.out
+'
+
+# we cheat and manually move active to inactive in these tests
+
+test_expect_success 'flux job eventlog works (inactive)' '
+        jobid=$(flux jobspec --format json srun -N1 hostname | flux job submit) &&
+        activekvsdir=$(flux job id --to=kvs-active $jobid) &&
+        inactivekvsdir=$(echo $activekvsdir | sed 's/active/inactive/') &&
+        flux kvs move ${activekvsdir}.eventlog ${inactivekvsdir}.eventlog &&
+	flux job eventlog $jobid > eventlog_c.out &&
+        grep submit eventlog_c.out
+'
+
+test_expect_success 'flux job eventlog works on multiple entries (inactive)' '
+        jobid=$(flux jobspec --format json srun -N1 hostname | flux job submit) &&
+        activekvsdir=$(flux job id --to=kvs-active $jobid) &&
+	flux kvs eventlog append ${activekvsdir}.eventlog foo &&
+        inactivekvsdir=$(echo $activekvsdir | sed 's/active/inactive/') &&
+        flux kvs move ${activekvsdir}.eventlog ${inactivekvsdir}.eventlog &&
+	flux job eventlog $jobid >eventlog_d.out &&
+	grep -q submit eventlog_d.out &&
+	grep -q foo eventlog_d.out
+'
+
+test_expect_success 'flux job eventlog works on multiple entries (active -> inactive)' '
+        jobid=$(flux jobspec --format json srun -N1 hostname | flux job submit) &&
+        activekvsdir=$(flux job id --to=kvs-active $jobid) &&
+        inactivekvsdir=$(echo $activekvsdir | sed 's/active/inactive/') &&
+        flux kvs move ${activekvsdir}.eventlog ${inactivekvsdir}.eventlog &&
+	flux kvs eventlog append ${inactivekvsdir}.eventlog foo &&
+	flux job eventlog $jobid >eventlog_e.out &&
+	grep -q submit eventlog_e.out &&
+	grep -q foo eventlog_e.out
+'
+
+#
+# job wait-event tests
+#
+
+test_expect_success NO_CHAIN_LINT 'flux job wait-event works' '
+        jobid=$(flux jobspec --format json srun -N1 hostname | flux job submit)
+        flux job wait-event $jobid foobar &
+        waitpid=$! &&
+        wait_lookups_nonzero &&
+        wait_watcherscount_nonzero primary &&
+        kvsdir=$(flux job id --to=kvs-active $jobid) &&
+	flux kvs eventlog append ${kvsdir}.eventlog foobar &&
+        wait $waitpid
+'
+
+test_expect_success NO_CHAIN_LINT 'flux job wait-event --timeout works' '
+        jobid=$(flux jobspec --format json srun -N1 hostname | flux job submit)
+        ! flux job wait-event --timeout=1 $jobid foobar 2> wait_event.err &&
+        grep "wait-event timeout" wait_event.err
+'
+
+test_expect_success NO_CHAIN_LINT 'flux job wait-event hangs on no event' '
+        jobid=$(flux jobspec --format json srun -N1 hostname | flux job submit)
+        ! run_timeout 3 flux job wait-event $jobid foobar
+'
```

Comment:
The 3s timeout here seems excessive, maybe just left over debug value?

In fact shouldn't .5 or even .2 seconds be enough? (run_timeout supports < 1s timeouts)  then the test overall will run 3x faster.

Reply:
I simply wanted a number that felt "timeout like", but you're right, it need not be larger.

Reply:
Yeah, figured it was something like that. The drawback is that we *always* pause for the timeout there.. so we want it to be as long as necessary and no longer ;-)